### ISSUE TITLE
Mask refinement instructions.

### ISSUE THREAD

**Issue by leftler** [shaoanlu/faceswap-GAN#68]
The instructions for mask refinement appear to be out of date
> ## Tips for mask refinement (optional after >15k iters)
> In Define loss function, change
> ```
> def define_loss(...)
> ...
> mixup = lam * concatenate([real, distorted]) + (1 - lam) * concatenate([fake, distorted])
> loss_G += K.mean(K.abs(fake_rgb - real))
> fake_sz224 = tf.image.resize_images(fake_rgb, [224, 224])
> ...
> # cell below
> loss_GA += 3e-3 * K.mean(K.abs(mask_A))
> loss_GB += 3e-3 * K.mean(K.abs(mask_B))
> ```
> to
> ```
> def define_loss(...)
> ...
> mixup = lam * concatenate([real, distorted]) + (1 - lam) * concatenate([fake, distorted])
> loss_G += K.mean(K.abs(fake - real))
> fake_sz224 = tf.image.resize_images(fake, [224, 224])
> ...
> # cell below
> loss_GA += 1e-3 * K.mean(K.square(mask_A))
> loss_GB += 1e-3 * K.mean(K.square(mask_B))
> ```
> If this gives better mask generation, then keep training.

There is no code that matches the "from" code. The closest I could find is is the code in the loss function surrounded by `# ==========` (which I figure is why you put it there) but it does not exactly match the 3 sections 

```
# ==========
mixup = lam * concatenate([real, distorted]) + (1 - lam) * concatenate([fake, distorted])
# ==========

# ==========  
loss_G += K.mean(K.abs(fake_rgb - real))
loss_G += K.mean(K.abs(fake_sz64 - tf.image.resize_images(real, [64, 64])))
# ==========

# ==========
fake_sz224 = tf.image.resize_images(fake_rgb, [224, 224]) 
fake_sz224 = Lambda(preprocess_vggface)(fake_sz224)
# ==========  
```

The thing I am unsure of is what is done with `loss_G += K.mean(K.abs(fake_sz64 - tf.image.resize_images(real, [64, 64])))`, Does that line need to be changed to anything new?

Also, side note, you resize the image to `[64, 64]` in the sz128 notebook. Is that intentional or should that be `[128, 128]`?

**Comment by shaoanlu** [comment]
We only change variable `fake_rgb` to `fake` for mask refinement. By doing this we allow loss gradient to flow through both the alpha mask and bgr channel layer (instead of bgr channel only).
